src/server/common/upload-status.js

```diff
@@ -0,0 +1,21 @@
+const UploadStatus = {
+  Initiated: 'initiated',
+  Quarantined: 'quarantined',
+  Scanned: 'scanned',
+  Delivered: 'delivered',
+  Acknowledged: 'acknowledged'
```

Comment:
FYI, the convention in js for uppercase first letter is to signify a class. As these are strings I would probably lowercase all of them and the wrapper `uploadStatus` so all are camelCase

Reply:
Was getting it mostly from https://www.sohamkamani.com/javascript/enums/ so that source may be a rebel.

Reply:
Since these are enums constants it seems odd to lowercase them but will do.